isyfact-standards-referenzimplementierung-doc/modules/ROOT/pages/basis/Referenzimplementierung-Architektur-Durchstich.adoc

```diff
@@ -256,35 +256,34 @@ der entsprechenden Java-Interfaces erfolgen.
 | Alle Awf-Klassen sind mit '@Service' annotiert.
 
 | de.bund.bva.isyfact.antrag.
-  core.daten.AntragBo
+core.daten.AntragBo
 | Anwendungskern und Service-Fassade arbeiten beide mit Business-Objekten
-  und tauschen ausschließlich Business-Objekte untereinander aus.
+und tauschen ausschließlich Business-Objekte untereinander aus.
 
   Die Persistenzschicht hingegen arbeitet mit Entitäten.
   Das bedeutet, dass für den Datenaustausch zwischen Persistenzschicht und Anwendungskern
   Entitäten und Business-Objekte ineinander umgewandelt werden müssen.
-
-|
+| /
 | de.bund.bva.isyfact.antrag.
   core.daten.AntragBoMapper
 | Diese Klasse enthält Konverter-Methoden
 
-  - für die Umwandlung von Antrag-Entitäten in entsprechende Bo-Objekte und
+- für die Umwandlung von Antrag-Entitäten in entsprechende Bo-Objekte und
 
-  - für die Umwandlung von Antrag-Bo-Objekten in entsprechende Entitäten
-|
+- für die Umwandlung von Antrag-Bo-Objekten in entsprechende Entitäten
```

Comment:
Aufzählungen werden in adoc Files mit * initialisiert
```suggestion
* für die Umwandlung von Antrag-Bo-Objekten in entsprechende Entitäten
```

Reply:
done